README.md

```diff
@@ -1,6 +1,7 @@
 # rust-qr-reader - Find and read QR-Codes
 [![documentation](https://docs.rs/rqrr/badge.svg)](https://docs.rs/rqrr/)
 [![Build Status](https://travis-ci.com/WanzenBug/rqrr.svg?branch=master)](https://travis-ci.com/WanzenBug/rqrr)
```

Comment:
You can remove this, this is no longer active.

Reply:
I applied the request. I also remove `.travis.yml`.